### PR TITLE
removes unneeded excess shotgun sprites

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

removes the double-barrel shotgun reskins.
## Why It's Good For The Game

it's a bunch of unneeded sprite debt... and does like anyone use these?
## Changelog
:cl: axietheaxolotl / Viro
image: removes the double-barrel shotgun reskins.
/:cl:

### CODE REVIEW

**Comment by DAKKA-WAAAGH** [comment]
I use these every round, bit fucked up to remove them! I choose Rosewood or Dark Red Finish often.

**Comment by NamelessFairy** [comment]
Wouldn't it be better to grayscale/gags these if the goal is to remove unneeded sprites?

**Comment by carlarctg** [comment]
Literally why lol

If someone resprites the shotgun but doesn't feel like doing every recolor sure

This 'sprite debt' is a totally made up concept. The sprites work perfectly well

**Comment by axietheaxolotl** [comment]
> Wouldn't it be better to grayscale/gags these if the goal is to remove unneeded sprites?

greyscale and gags sucks and i feel the need not to elaborate.



> Literally why lol
> 
> If someone resprites the shotgun but doesn't feel like doing every recolor sure
> 
> This 'sprite debt' is a totally made up concept. The sprites work perfectly well

because then one wil look good and the others will not. sprite debt is a real thing and I refuse to argue with someone who can't recognise the validity of it as a concern.

**Comment by Helg2** [comment]
> because then one wil look good and the others will not. sprite debt is a real thing and I refuse to argue with someone who can't recognise the validity of it as a concern.

Are you twing of viro or what

**Comment by axietheaxolotl** [comment]
> > because then one wil look good and the others will not. sprite debt is a real thing and I refuse to argue with someone who can't recognise the validity of it as a concern.
> 
> Are you twink of viro or what

what?

**Comment by Helg2** [comment]
> what?

Nevermind, your reasoning is retardish, no one cares about sprite debt, you are removing feature from the game and propose nothing in exchange.

**Comment by axietheaxolotl** [comment]
> > what?
> 
> Nevermind, your reasoning is retardish, no one cares about sprite debt, you are removing feature from the game and propose nothing in exchange.

retardish is a word.

**Comment by Rex9001** [comment]
I like recoloring muh shotun….

**Comment by Helg2** [comment]
> retardish isnt a word.

Who asked

**Comment by axietheaxolotl** [comment]
> Besides the fact that it is. Who asked
you know who cares about sprite debt? ...spriters, and people who have to deal with it, which you aren't so please be quiet. Also a couple of reskins is not a feature, this isn't removing nanites.

**Comment by Likteer** [comment]
Github desperately needs a nerd reaction.

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 1]
```diff
@@ -262,15 +262,7 @@
 	slot_flags = ITEM_SLOT_BACK
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/dual
 	sawn_desc = "Omar's coming!"
-	obj_flags = UNIQUE_RENAME
 	rack_sound_volume = 0
```
Comment (Thunder12345) [inline code/modules/projectiles/guns/ballistic/shotgun.dm:265]: This is for renaming, not related to the reskin.

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 2]
```diff
@@ -262,15 +262,7 @@
 	slot_flags = ITEM_SLOT_BACK
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/dual
 	sawn_desc = "Omar's coming!"
-	obj_flags = UNIQUE_RENAME
```
Comment (Thunder12345) [inline code/modules/projectiles/guns/ballistic/shotgun.dm:265]: This is for changing the name/description with a pen, unrelated to the visual reskin.

**Review by MrMelbert — CHANGES_REQUESTED** [review]
GAGS is a great way of reducing sprite debt, one of the skins is already grey you can go from there.
And you don't need to complain about them being ugly since "no one uses them" and you'll never see them in game

**Comment by necromanceranne** [comment]
You also forgot about the inhands and worn sprites.

**Comment by ChungusGamer666** [comment]
>Greyscale and gags suck
>Nukes all of the colors

powerful

**Comment by axietheaxolotl** [comment]
> You also forgot about the inhands and worn sprites.

unless i'm blind as fuck (i could be), there's no inhand or worn sprites for the reskins (which only emboldens my point that they're useless lol)

**Comment by axietheaxolotl** [comment]
> > Greyscale and gags suck
> > Nukes all of the reskins instead
> 
> powerful

i have no issue with reskins, reskins are kino. the problem comes when it's bad/useless reskins. like these.

**Comment by Thunder12345** [comment]
> > You also forgot about the inhands and worn sprites.
> 
> unless i'm blind as fuck (i could be), there's no inhand or worn sprites for the reskins (which only emboldens my point that they're useless lol)

Worn back sprites

**Comment by borb197** [comment]
> > > Greyscale and gags suck
> > > Nukes all of the reskins instead
> > 
> > 
> > powerful
> 
> i have no issue with reskins, reskins are kino. the problem comes when it's bad/useless reskins. like these.

all of the wallening sprites are bad/useless reskins, but the only sprites we lose are the only ones that nobody want removed

**Comment by ChungusGamer666** [comment]
> i have no issue with reskins, reskins are kino. the problem comes when it's bad/useless reskins. like these.

you and i both know this shit is subjective as hell and you're using your personal, i am a fan of thes

**Comment by Twaticus** [comment]
> > Wouldn't it be better to grayscale/gags these if the goal is to remove unneeded sprites?
> 
> greyscale and gags sucks and i feel the need not to elaborate.

damn bro thats crazy

**Pull request closed without merging**